src/main/java/Main.java

```diff
@@ -1,9 +1,20 @@
+import java.util.Scanner;
+
 /**
  * Created by iyasuwatts on 10/17/17.
  */
 public class Main {
 
     public static void main(String[] args){
 
+        Scanner scanner1 = new Scanner(System.in);
+        System.out.println("Hello! Please put in a number");
+        int n = scanner1.nextInt();
+
+        for (int i = 0; i < n; i++){
+            System.out.println(i);
```

Comment:
you'll need to rework this to print the sum of the ints from 0 to n... not a list of ints.